asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryProtectionReport.tex

```diff
@@ -0,0 +1,193 @@
+\vspace{0.4cm}
+{\huge\bfseries \thetitle} \\
+\vspace{0.2cm}
+{\LARGE \thesubtitle} \\
+\vspace{2.2cm}
+\ifdefined \theauthor
+\par Prepared by: \\
+\theauthor \\
+\theauthorcontact \\
+\fi
+\thedate \\
+\vfill
+\theteamwebsite \\
+\theteamphone
+\end{titlepage}
+
+% Main Matter
+\tableofcontents
+\newpage
+%\listoffigures % <-------------- uncomment for list of figures
+%\listoftables % <-------------- uncomment for list of tables
+
+\section{Overview}
+
+\subsection{5.2.E.1: Battery Approval Form}
+
+See attached form
+
+\includepdf[pages={-}]{forms/battery_approval_form.pdf}
+
+\subsection{5.2.E.2: Battery Pack Configuration}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Cells per module}    & 36   \\ \hline
+\textbf{Modules per string}  & 36   \\ \hline
+\textbf{Strings in parallel} & 1    \\ \hline
+\textbf{Total cell count}    & 1296 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.3: Over temperature set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Over temperature set point (charge) (C)}    & 45 \\ \hline
+\textbf{Over temperature set point (discharge) (C)} & 60 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.4: Under voltage set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Under voltage set point (V)}    & 2.5 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.5: Over voltage set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Over voltage set point (V)}    & 4.2 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.6: Over current set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Over current set point (A)}    & 110 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.7: BPS Block Diagram}
+
+\textbf{TODO: add legend}
+
+\includegraphics[width=1\textwidth]{./figures/msxii-electrical-battery-box-block-diagram.png}
+
+\subsection{5.2.E.8: BPS Operation}
+
+The BPS will monitor for each of the fault conditions on each iteration of its
+\texttt{main} loop. We use the \texttt{LTC6804-1} in a daisy-chain
+configuration to read the voltages and temperatures of each battery module, and
+we use an ADC and a shunt resistor to measure the current.
+
+If it detects a fault condition, it will raise a BPS fault error, which is
+broadcast on the CAN bus. This message is processed by our
+\texttt{power distribution} and \texttt{driver controls} boards. It will also
+drive the two pack fans.
+
+The only way to clear a fault will be to power off the car, which will result
+in the car undergoing the proper power-sequencing steps.
+
+\subsection{5.2.E.9: Rendering Firmware Static}
+
+We will seal the programming header of the BPS microcontroller, which will
+make it impossible to reprogram the firmware.
+
+\subsection{5.2.E.10: Driver dash and BPS fault indicator}
+
+\subsubsection{Driver Dash}
+When the \texttt{Lights} board receives the BPS fault message, it turns on the
```

Comment:
Should this be in a different subsubsection?

Reply:
no, this is part of the same regulation

Reply:
Right, but it seems to be about the strobe light instead of the driver dash.